packages/shared/scope-hoisting/src/utils.js

```diff
@@ -314,11 +311,143 @@ export function convertBabelLoc(loc: ?BabelSourceLocation): ?SourceLocation {
     filePath: path.normalize(filename),
     start: {
       line: start.line,
-      column: start.column,
-    },
-    end: {
-      line: end.line,
-      column: end.column,
+      column: start.column + 1,
     },
+    // - Babel's columns are exclusive, ours are inclusive (column - 1)
+    // - Babel has 0-based columns, ours are 1-based (column + 1)
+    // = +-0
+    end,
   };
 }
+
+export function getExportNamespaceExpression(
```

Comment:
Are we not keeping the namespaces generated by hoist anymore?

Reply:
If only I could remember why I did this 🙈

Reply:
So I think the reason here was that exports in shared bundles wouldn't be shaken because the namespace objects always contains all exports, not just the used ones. 

So if you have a shared bundle that contains this asset
```js
export const a = 2;
export const b = 4;
```

and `b` is never actually used, this synthetic namespace objects contains `{a: $id$export$a}` so that `b` can be `shake()`n.